### PR TITLE
Season 2 Finale

### PR DESCRIPTION
**Pull request by mre**

### CODE REVIEW

**Comment by mre** [comment]
@prscheduler 08/08/2024T16:00

File: content/podcast/s02e08-season-finale/index.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+If you missed any of the previous episodes, you can find them here:
+
+- [Episode 1: curl with Daniel Stenberg](/podcast/s02e01-curl)
+- [Episode 2: AMP with Carter Schultz](/podcast/s02e02-amp)
+- [Episode 3: Thunderbird with Brendan Abolivier](/podcast/s02e03-thunderbird)
+- [Episode 4: Matic with Eric Seppanen](/podcast/s02e04-matic)
+- [Episode 5: OxidOS with Alexandru Radovici](/podcast/s02e05-oxidos)
+- [Episode 6: Fusion Engineering with Jakub Valtar](/podcast/s02e06-fusion-engineering)
+- [Episode 7: System76 with Jeremy Soller](/podcast/s02e07-system76)
```
Comment (M3t0r) [inline content/podcast/s02e08-season-finale/index.md]: We might have to change these :sweat_smile: 

There are some links at the end of the doc.

When these are updated, also update them in Let's Cast show notes!

File: content/podcast/s02e08-season-finale/index.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+title = "Season 2 - Finale"
+date = 2024-08-08
+template = "episode.html"
+draft = false
+[extra]
+season = "02"
+episode = "08"
+series = "Podcast"
++++
+
```
Comment (M3t0r) [inline content/podcast/s02e08-season-finale/index.md:11]:
```suggestion

<div><script id="letscast-player-7ae4f95b" src="https://letscast.fm/podcasts/rust-in-production-82281512/episodes/rust-in-production-ep-15-season-finale/player.js?size=s"></script></div>

```

**Review by M3t0r — CHANGES_REQUESTED** [review]

File: content/podcast/s02e08-season-finale/index.md (hunk 1 above)
Comment (mre) [reply content/podcast/s02e08-season-finale/index.md]: done

**Pull request merged**